auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -187,13 +191,12 @@ public String gotoPage(@PathVariable("page") String page) {
     }
 
     @RequestMapping(value = "/admin/sessions", method = RequestMethod.GET)
-    public
-    @ResponseBody
+
+    public @ResponseBody
     RestWrapper list(HttpSession session) {
         //reset session timeout
         SessionHolder.resetSessionTimeout(session);
-        RestWrapper restWrapper = new RestWrapper(SessionHolder.getAllAuthResults(), RestWrapper.OK);
-        return restWrapper;
+        return new RestWrapper(SessionHolder.getAllAuthResults(), RestWrapper.OK);
     }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "logout". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)